opensrp-chw-hiv/src/main/res/layout/activity_base_hiv_profile.xml

```diff
@@ -188,6 +188,21 @@
                     android:paddingStart="0dp"
                     android:paddingEnd="0dp"
                     android:textAppearance="@style/TextAppearance.AppCompat.Small" />
+                <org.smartregister.view.customcontrols.CustomFontTextView
```

Comment:
Let's change the default visibility of this to GONE so that it does not affect other clients on CBHS and HTS services who do not have the CTC Numbers.